README.md

```diff
@@ -10,6 +10,15 @@ files.
 
 This repo also powers the Scala syntax highlighting on GitHub. (It is vendored in [github/linguist](https://github.com/github/linguist).)
 
+### Team
+
+The current maintainers (people who can merge pull requests) are:
+
+- Olafur Pall Geirsson - [`@olafurpg`](https://github.com/olafurpg)
+- Nicolas Stucki- [`@nicolasstucki`](https://github.com/nicolasstucki)
+- Vitalii Voloshyn - [`@PanAeon`](https://github.com/PanAeon)
```

Comment:
@PanAeon I inferred this name from your gmail address, please correct me if I got it wrong 😅